lms/djangoapps/teams/views.py

```diff
@@ -330,7 +330,13 @@ def get(self, request):
             result_filter.update({'topic_id': request.QUERY_PARAMS['topic_id']})
 
         if text_search and CourseTeamIndexer.search_is_enabled():
-            search_engine = CourseTeamIndexer.engine()
+            try:
+                search_engine = CourseTeamIndexer.engine()
+            except ElasticSearchConnectionError:
```

Comment:
@adampalay We are logging further down the stack.  Whenever we create an instance of `ElasticSearchConnectionError` regardless of what happens to it upstream.  Do you still think we need it?